gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoints-group/configuration/api-endpoints-group-configuration.component.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Subject } from 'rxjs';
+import { map, takeUntil } from 'rxjs/operators';
+
+import { ConnectorService } from '../../../../../../services-ngx/connector.service';
+
+@Component({
+  selector: 'api-endpoints-group-configuration',
+  template: require('./api-endpoints-group-configuration.component.html'),
+  styles: [require('./api-endpoints-group-configuration.component.scss')],
+})
+export class ApiEndpointsGroupConfigurationComponent implements OnInit, OnDestroy {
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+
+  @Input() groupConfigurationControl: FormControl;
+
+  public schemaForm: unknown;
+
+  constructor(private readonly connectorService: ConnectorService) {}
+
+  ngOnInit(): void {
+    this.connectorService
+      .list(true)
+      .pipe(
+        map((connectors) => {
```

Comment:
Not `tap`? I thought `map` for rxjs is more to change value of the initial value?